tes-adapter/src/main/java/com/epam/pipeline/tesadapter/service/TesTaskServiceImpl.java

```diff
@@ -57,8 +68,32 @@ public TesCreateTaskResponse submitTesTask(TesTask body) {
     }
 
     @Override
-    public TesListTasksResponse listTesTask() {
-        return new TesListTasksResponse();
+    public TesListTasksResponse listTesTask(String namePrefix, Long pageSize, String pageToken, TaskView view) {
+        return new TesListTasksResponse((StringUtils.isNotEmpty(namePrefix) ?
```

Comment:
this way of writing would be easy to read
```
final listedTasks = (StringUtils.isNotEmpty(namePrefix) 
        ? searchRunsWithNamePrefix(namePrefix, pageSize, pageToken)
        : filterRunsWithOutNamePrefix(pageSize, pageToken)
).stream()
 .map(pipelineRun ->
                taskMapper.mapToTesTask(
                            pipelineRun, 
                            Optional.ofNullable(view).orElse(DEFAULT_TASK_VIEW)
                ))
 .collect(Collectors.toList())
return new TesListTasksResponse(listedTasks);
```

Reply:
fixed